front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -8,24 +8,29 @@ export function ConversationMessage({
   children,
   name,
   pictureUrl,
+  buttons,
 }: {
   children?: React.ReactNode;
   name: string | null;
   pictureUrl?: string | null;
+  buttons?: React.ReactNode[];
 }) {
   return (
     <div className="flex w-full flex-row gap-4">
       <div className="flex-shrink-0">
         <Avatar visual={pictureUrl} name={name || undefined} size="sm" />
       </div>
       <div className="min-w-0 flex-grow">
-        <div className="flex-col gap-4">
+        <div className="flex flex-col gap-4">
           <div className="text-sm font-medium">{name}</div>
           <div className="min-w-0 break-words text-base font-normal">
             {children}
           </div>
         </div>
       </div>
+      <div className="ite flex flex-col items-start gap-2 sm:flex-row">
```

Comment:
lot of assumptions on what is happening below no?

Reply:
Yes. In general, it would be a bad thing for random children
Here, we explicitley pass a list of buttons to the component and it displays it properly => in that case it is justified I think